source/validate_cfg.cpp

```diff
@@ -173,21 +173,23 @@ string ConstructErrorString(const Construct& construct,
 // |case_fall_through|. Returns SPV_ERROR_INVALID_CFG if the case construct
 // headed by |target_block| branches to multiple case constructs.
 spv_result_t FindCaseFallThrough(
-    const ValidationState_t& _, const BasicBlock* target_block,
+    const ValidationState_t& _, BasicBlock* target_block,
     uint32_t* case_fall_through, const BasicBlock* merge,
-    const std::unordered_set<uint32_t>& case_targets) {
-  std::vector<const BasicBlock*> stack;
+    const std::unordered_set<uint32_t>& case_targets, Function* function) {
+  std::vector<BasicBlock*> stack;
   stack.push_back(target_block);
   std::unordered_set<const BasicBlock*> visited;
+  bool target_reachable = target_block->reachable();
```

Comment:
Better for this to be const.  It took me a few seconds to see it never changed since target_block never changes.